test/Microsoft.TemplateEngine.Core.UnitTests/StubSettingsLoader.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.TemplateEngine.Abstractions;
+using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Abstractions.Mount;
+
+namespace Microsoft.TemplateEngine.Core.UnitTests
+{
+    internal class StubSettingsLoader : ISettingsLoader
+    {
+        public IComponentManager Components => throw new NotImplementedException();
+
+        public IEngineEnvironmentSettings EnvironmentSettings => throw new NotImplementedException();
+
+        public IEnumerable<MountPointInfo> MountPoints => throw new NotImplementedException();
+
+        public void AddMountPoint(IMountPoint mountPoint)
+        {
+            throw new NotImplementedException();
```

Comment:
If we're adding a new mock, we should go all the way here, others that are trying to test extensions to the engine wouldn't have a good experience with all the throws